server/database/schema.sql

```diff
@@ -13,17 +13,16 @@ CREATE TABLE ingredient (
     id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
     name VARCHAR(255) NOT NULL,
     category VARCHAR(255) NOT NULL,
-    image TEXT NOT NULL,
+    image VARCHAR(255) DEFAULT '../../client/public/images/Logo_foodtastics.png',
```

Comment:
Replace this by removing everything before /image , path must be absolute and not relative.
Example : /images/logos/my logo.jpg

Reply:
Done :)